helper-cli/src/main/kotlin/commands/ImportScanResultsCommand.kt

```diff
@@ -57,7 +58,7 @@ internal class ImportScanResultsCommand : CliktCommand(
 
         ids.forEach { id ->
             ortResult.getScanResultsForId(id).forEach { scanResult ->
-                scanResultsStorage.add(id, scanResult)
+                scanResultsStorage.add(id, scanResult.copy(rawResult = EMPTY_JSON_NODE))
```

Comment:
Commit message: I had difficulties understanding what "which enables it's import" means. Could make this clearer? For example: "The scan results in the `OrtResult` do not contain the raw results, but the `ScanResultsStorage` requires that the raw results are not `null` when adding scan results. Therefore set the raw results to empty JSON nodes before adding them to the local storage."

Reply:
Done.